digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/LogUtils.scala

```diff
@@ -0,0 +1,34 @@
+package fi.liikennevirasto.digiroad2
```

Comment:
Pitäis varmaan katsoa tuota meidän buildia, että miten pitää muuttaa, että saadaan Digiroad2-oracle moduulin luokkia hyödynnettyä geo moduulissa. Tästä voisi tiketin kirjata elinkaaren hallintaan. Duplikaatti luokka siihen asti ok mielestäni

Reply:
joo, itsekin vähän miettinyt jotakin uutta moduuli rakennetta tai sen kokonaan purkamista, Toi utils homma vaatis muutenkin vähän enemmän organisointia kun sekalainen kooste tavaraa nyt.